common/v2/services/EthService/nodes/web3/client.ts

```diff
@@ -1,6 +1,6 @@
+import { RPCClient } from '../rpc';
 import { JsonRpcResponse, RPCRequest } from '../rpc/types';
```

Comment:
Import can be simplified and combined with the one above.

Reply:
Done. 

This is one of the trickiest issues I encountered.
During the transforms, Babel is suppose to remove the all ts annotations.
When we import a type with a JS object or function, at times, Babel will remove the exported type from the original file but leave the import thus leading to warnings like `RPCRequest` not found.

Originally I used this work around before moving the file.
Changing back to `../rpc` seems to work now.
Any insights are welcome!